webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+    entry:{
+      bundle: './src/assets/index.js',
```

Comment:
hmmm I get it! But I guess you can set more then one file as entry, can'r you?

Reply:
yea it's works like: 
entry:{
      bundle: './src/assets/index.js',
        main: './src/assets/index.js',
          otherName: './src/assets/index.js',
}

Reply:
then you don't need the index.js!

Reply:
why not? it says that the entry file(the index.js) will be named bundle by webpack.config
and the index.js it's the file i'am doing the export to the Main.js

Reply:
if you're saying i don't need it.
the path the file have to do it's for the Main.js ?🤔
like:
entry:{
bundle: './src/assets/Main.js',
}

Reply:
yep it'll be renamed, then if you use xyz.js it will be build using index.js